### PR TITLE
2022-09-12 TSC meeting minutes

### PR DESCRIPTION
**Pull request by weierophinney**
- Provides minutes for the 2022-09-12 TSC meeting
- Resets the agenda for the 2022-10-03 TSC meeting

### CODE REVIEW

File: meetings/minutes/2022-09-12-TSC-Minutes.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+## Agenda
+
+### Default git branching for new majors
+
+When creating a new major, the relevant branch is created; it is also set as the default branch for the repository. However, there can be a significant gap between branch creation and the first release of that major version.
+
+Contributors and renovate, will attempt to make bugfixes to this new major when they equally apply to the previous (current latest release) major.
+
+I (ed: George Steel) propose that the default branch should only be changed to the new major branch, once a release has been tagged for that new major.
```
Comment (internalsystemerror) [inline meetings/minutes/2022-09-12-TSC-Minutes.md:26]: just to point out to save @gsteel from any future hassle, this was my submission, I forgot to add my name in the original PR, my bad

File: meetings/minutes/2022-09-12-TSC-Minutes.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+This had no debate; anybody who has had to maintain a repo that has an unreleased new major branch has run into the issue.
+Matthew noted that we need automatic-releases to create the new minor branch when a new major branch already exists, as that's the root cause of the problem.
+Aleksei noted that the logic should be "if the newly created branch version is higher than the current default branch, set the default to the new branch; otherwise, don't."
+
+Matthew volunteered to create an issue on the automatic-releases repo to request this.
+
+### Security only packages and dependency updates
+
+Is this intention for security only packages to eventually be abandoned? If no, then should we (I) look at ways to allow renovate to keep these up to date, if only to give us warning when the package may actually need maintenance. Currently any PR would be automatically closed, including the "Configure Renovate" PRs.
```
Comment (internalsystemerror) [inline meetings/minutes/2022-09-12-TSC-Minutes.md:38]: forgot to add my name in the original PR here too, sorry

File: meetings/minutes/2022-09-12-TSC-Minutes.md (hunk 1 above)
Comment (weierophinney) [reply meetings/minutes/2022-09-12-TSC-Minutes.md]: Updated!

File: meetings/minutes/2022-09-12-TSC-Minutes.md (hunk 2 above)
Comment (weierophinney) [reply meetings/minutes/2022-09-12-TSC-Minutes.md]: Added attribution

**Review by gsteel — APPROVED** [review]

**Pull request merged**